### PR TITLE
Add email & time id to each comment and sort comments in desc. order of time

### PR DESCRIPTION
**Pull request by dihim**
Added user email and the time they posted to each comment and sorted the array of comments to be displayed by time

- Identified current user logged in DataServlet & added user email and time info to each comment entity
- Sorted the array list of comments in DataServerlet by time

TESTED
I logged in with email and posted a comment and the comment had my email in front of the text with the comment text in the middle ending in the server time it was posted. I posted two more comments which followed the same format but also displayed the most recent comment at the top with other comments in descending order of time

### CODE REVIEW

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      String urlToRedirectToAfterUserLogsIn = "/forum.html";
+      String loginUrl = userService.createLoginURL(urlToRedirectToAfterUserLogsIn);
+      HashMap<String, String> userInfo = new HashMap<String, String>();
+      userInfo.put("loggedIn", new Boolean(loggedIn).toString());
+      userInfo.put("loginUrl", loginUrl);
+      response.getWriter().println(convertToJsonUsingGson(userInfo));
+    }
+  }
+
+  private String convertToJsonUsingGson(HashMap<String, String> userInfo) {
```
Comment (ElanHR) [inline portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java:59]: ditto

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+import java.util.HashMap;
+import com.google.gson.Gson;
+
+
+
+@WebServlet("/login-user")
+public class LoginServlet extends HttpServlet {
+
+  @Override
+  public void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
```
Comment (ElanHR) [inline portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java:33]: Please add a brief comment describing its behavior.

**Review by ElanHR — CHANGES_REQUESTED** [review]

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java (hunk 2 above)
Comment (dihim) [reply portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java:33]: Gotcha!

File: portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java (hunk 1 above)
Comment (dihim) [reply portfolio/src/main/java/com/google/sps/servlets/LoginServlet.java:59]: Thanks, will add

**Review by ElanHR — APPROVED** [review]

**Pull request merged**